### PR TITLE
[SCB-1369] Save the terminated transaction data to elasticsearch

### PR DESCRIPTION
**Pull request by coolbeevip**
Transaction persistence to Elasticsearch
Add transaction query API based on Elasticsearch
Refactoring Acceptance Test merge multiple cucumber features

### CODE REVIEW

File: alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/repository/NoneTransactionRepository.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NoneTransactionRepository implements TransactionRepository {
+
+  private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  @Override
+  public void send(GlobalTransaction transaction) {
+    LOG.warn("Please configure alpha.feature.akka.transaction.repository.type=elasticsearch to use elasticsearch to persist transaction data.");
```
Comment (WillemJiang) [inline alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/repository/NoneTransactionRepository.java]: I think we can add a line in the warning message "NoneTransactionRepostory is set."

File: alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/repository/elasticsearch/GlobalTransactionDocument.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package org.apache.servicecomb.pack.alpha.fsm.repository.elasticsearch;
```
Comment (WillemJiang) [inline alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/repository/elasticsearch/GlobalTransactionDocument.java]: ASL header.

Comment (coolbeevip) [reply alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/repository/elasticsearch/GlobalTransactionDocument.java]: Oh, it has been corrected

File: alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/repository/NoneTransactionRepository.java (hunk 1 above)
Comment (coolbeevip) [reply alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/repository/NoneTransactionRepository.java]: Good advice

**Pull request merged**